src/screens/UserPortal/Events/__mocks__/EventHeaderMock.tsx

```
@@ -0,0 +1,55 @@
+  viewType,
+  handleChangeView,
+  showInviteModal,
+}: Props) {
+  return (
+    <div>
+      <div data-testid="calendarEventHeader">
+        <div className="_calendar__controls">
+          <button
+            data-testid="selectViewType"
+            onClick={() => handleChangeView?.('MONTH')}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)